src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/TracePoint.kt

```diff
@@ -237,6 +265,16 @@ internal class MethodCallTracePoint(
     }
 }
 
+private fun List<String>.checkDefault(paramNames: List<String>, defaultValues: List<String>): List<String> {
```

Comment:
Let rename to `getNamedArguments` or something similar. 
Also please add kdoc explaining the function semantics, its parameters, and return value.